modules/hal_nordic/nrfs/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if(CONFIG_NRFS)
+    if(CONFIG_NRFS_LOCAL_DOMAIN)
```

Comment:
CMakeLists.txt in the hal/nordic/nrfs is not included.
```diff
diff --git a/modules/hal_nordic/nrfs/CMakeLists.txt b/modules/hal_nordic/nrfs/CMakeLists.txt
index a4edc6be513..eec370cf9b1 100644
--- a/modules/hal_nordic/nrfs/CMakeLists.txt
+++ b/modules/hal_nordic/nrfs/CMakeLists.txt
@@ -2,6 +2,8 @@
 # SPDX-License-Identifier: Apache-2.0
 
 if(CONFIG_NRFS)
+    add_subdirectory(${ZEPHYR_CURRENT_MODULE_DIR}/nrfs nrfs)
+
     if(CONFIG_NRFS_LOCAL_DOMAIN)
         zephyr_sources_ifdef(CONFIG_NRFS_BACKEND_IPC_SERVICE_EXTENDED
                             backends/nrfs_backend_ipc_service.c)
```

Reply:
CMakeLists are now completely reorganized.